### PR TITLE
revert: pixels, keep bleed

### PR DESCRIPTION
**Pull request by bouveng**
## Summary of Changes
Reverts pixels , restores screen emissive intensity, but keeps the bleed effect + tint.

Discord username: @bouveng

## Testing instructions
1, spawn
2, verify pixels is removed from PFD, ND, EICAS, MCDU and DCDU.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Review by davidwalschots — CHANGES_REQUESTED** [review]
Please remove the commented out code. Git will take care of keeping the history 😀.

**Comment by bouveng** [comment]
> Please remove the commented out code. Git will take care of keeping the history 😀.

Ok, right, that is true.

File: flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml [hunk 1]
```diff
@@ -505,7 +505,7 @@
                     <PART_ID>SCREEN_PFD1</PART_ID>
                     <CAMERA_TITLE>PFD</CAMERA_TITLE>
                     <!-- Pixelated screens needs more light -->
-                    <EMISSIVE_CODE>2</EMISSIVE_CODE>
+                    <!-- Pixels disabled for now -->
```
Comment (davidwalschots) [inline flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml:508]:
```suggestion
```

File: flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml [hunk 2]
```diff
@@ -606,7 +606,7 @@
                     <PART_ID>SCREEN_MFD1</PART_ID>
                     <CAMERA_TITLE>MFD</CAMERA_TITLE>
                     <!-- Pixelated screens needs more light -->
-                    <EMISSIVE_CODE>2</EMISSIVE_CODE>
+                    <!-- Pixels disabled for now -->
```
Comment (davidwalschots) [inline flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml:609]:
```suggestion
```

File: flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml [hunk 3]
```diff
@@ -650,7 +650,7 @@
                     <POTENTIOMETER>90</POTENTIOMETER>
                     <PART_ID>SCREEN_PFD2</PART_ID>
                     <!-- Pixelated screens needs more light -->
-                    <EMISSIVE_CODE>2</EMISSIVE_CODE>
+                    <!-- Pixels disabled for now -->
```
Comment (davidwalschots) [inline flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml:653]:
```suggestion
```

File: flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml [hunk 4]
```diff
@@ -667,7 +667,7 @@
                     <POTENTIOMETER>91</POTENTIOMETER>
                     <PART_ID>SCREEN_MFD2</PART_ID>
                     <!-- Pixelated screens needs more light -->
-                    <EMISSIVE_CODE>2</EMISSIVE_CODE>
+                    <!-- Pixels disabled for now -->
```
Comment (davidwalschots) [inline flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml:670]:
```suggestion
```

File: flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml [hunk 5]
```diff
@@ -795,7 +795,7 @@
                 <UseTemplate Name="ASOBO_GT_Component_Emissive_Gauge">
                     <NODE_ID>DYN_SCREEN</NODE_ID>
                     <!-- Pixelated screens needs more light -->
-                    <EMISSIVE_CODE>2</EMISSIVE_CODE>
+                    <!-- Pixels disabled for now -->
```
Comment (davidwalschots) [inline flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml:798]:
```suggestion
```

File: flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml [hunk 6]
```diff
@@ -881,7 +881,7 @@
                     <PART_ID>SCREEN_EICAS1</PART_ID>
                     <CAMERA_TITLE>ECAM</CAMERA_TITLE>
                     <!-- Pixelated screens needs more light -->
-                    <EMISSIVE_CODE>2</EMISSIVE_CODE>
+                    <!-- Pixels disabled for now -->
```
Comment (davidwalschots) [inline flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml:884]:
```suggestion
```

File: flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml [hunk 7]
```diff
@@ -898,7 +898,7 @@
                     <PART_ID>SCREEN_EICAS2</PART_ID>
                     <CAMERA_TITLE>ECAM</CAMERA_TITLE>
                     <!-- Pixelated screens needs more light -->
-                    <EMISSIVE_CODE>2</EMISSIVE_CODE>
+                    <!-- Pixels disabled for now -->
```
Comment (davidwalschots) [inline flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml:901]:
```suggestion
```

File: flybywire-aircraft-a320-neo/html_ui/CSS/A32NX_Display_Common.css [hunk 8]
```diff
@@ -54,7 +54,7 @@ text {
   z-index: 999;
   position: absolute;
   /* custom svg */
-  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 10 10'%3E%3Cpath d='M0 0 V10 Z H10 Z' stroke='%23000' stroke-width='3' fill='none'/%3E%3C/svg%3E");
+  /* pixels disabled for now */
```
Comment (davidwalschots) [inline flybywire-aircraft-a320-neo/html_ui/CSS/A32NX_Display_Common.css:57]:
```suggestion
```

File: src/instruments/src/Common/pixels.scss [hunk 9]
```diff
@@ -13,7 +13,7 @@
     background-position: 1px 0;
     background-color: rgba(80, 80, 80, 0.2);
   } @else {
-    background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 10 10'%3E%3Cpath d='M0 0 V10 Z H10 Z' stroke='%23000' stroke-width='#{$spacing}' fill='none'/%3E%3C/svg%3E");
+    /* pixels disabled for now */
```
Comment (davidwalschots) [inline src/instruments/src/Common/pixels.scss:16]:
```suggestion
```

File: src/instruments/src/DCDU/style.scss [hunk 10]
```diff
@@ -13,10 +13,7 @@
 }
 
 /* pixels for screen */
-@import "../Common/pixels";
-#panel:after {
-  @include pixels($checkered: true);
-}
+/* pixels disabled for now */
```
Comment (davidwalschots) [inline src/instruments/src/DCDU/style.scss:16]:
```suggestion
```

**Review by davidwalschots — CHANGES_REQUESTED** [review]

File: src/instruments/src/DCDU/style.scss [hunk 11]
```diff
@@ -13,10 +13,7 @@
 }
 
 /* pixels for screen */
```
Comment (davidwalschots) [inline src/instruments/src/DCDU/style.scss:15]:
```suggestion
```

**Review by davidwalschots — APPROVED** [review]

**Comment by bouveng** [comment]
Await alternative solution from @2hwk , this PR might not be needed.

**Comment by 2hwk** [comment]
See #5483, this should be unnecessary now.

**Pull request closed without merging**